langfuse/client.py

```diff
@@ -190,8 +189,8 @@ def create_generation():
                     self.log.exception(e)
                     raise e
 
-            self.task_manager.add_task(new_generation_id, create_generation)
             self.task_manager.add_task(new_trace_id, create_trace)
+            self.task_manager.add_task(new_generation_id, create_generation)
```

Comment:
Ordering does not matter, but more logical to have it this way

Reply:
Yes